website/content/maintainers/community/vendor-neutrality.md

```diff
@@ -0,0 +1,91 @@
+
+### Websites
+
+* Please refer to the CNCF [website guidelines](https://github.com/cncf/foundation/blob/main/website-guidelines.md)
+
+### Blogs
+
+* Blogs may mention a vendor’s name as it relates to specific open source project, project deployments, adoption paths, their hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, but it shouldn’t feel like an advertisement for the services or company
+* Blogs hosted on project websites should be about the open source project, any references to enterprise offerings should point to the enterprise page of the project that features all vendors equally rather than linking to one specific vendor
+* Project blogs should not favor one vendor’s solution over another
+* Project blogs should keep other CNCF projects in a positive light
+
+### Social media
+
+* Projects can choose whether they want to repost content from vendors, however the rules of engagement should be consistent across vendors and ideally written down so they are clear to everyone
+* Post/re-posts from project handles should not have content that disparages different vendors or CNCF projects
+
+### Messaging platforms, such as Slack
+
+* It is up to the project whether vendors can post in their messaging channels, however the rules of engagement should be consistent across vendors and ideally written down so they are clear to everyone
+* Vendors should avoid channel spamming or solicitation through private messages
+
+## Hosting
+
+Wherever possible, community meetings, events, resources, and infrastructure should be hosted on resources belonging to the CNCF, or on other neutral, 3rd-party resources (like GitHub).  Good resources to use include:
+
+* CNCF Zoom or Bevy for meetings
+* GitHub or Netlify static site hosting for website and docs
+* CNCF cloud credits for testing
+* CNCF-shared YouTube account for videos
+* GitHub issues & project boards for tickets and roadmap items
+
+Depending on the status of your project, you may not have all resources available to you. In other cases, projects may need a resource that's not normally supported by the CNCF. If project sponsors have to self-host resources, they should try to still separate resources affiliated with the CNCF project from resources attached to their products. For example, if a project uses Trello, there should be a separate Trello account for the projects from any sponsor's product boards. File a [service desk ticket](https://cncfservicedesk.atlassian.net/servicedesk/customer/portal/1) to see whether unique services needed by the project could be funded by the CNCF.
+
+Public events for the project must also be handled in a vendor-neutral way. Even in cases where the CNCF events staff is not engaged, all potential vendors must have an equal opportunity to sponsor and participate in the event. For this reason, hosting an project-branded event at any specific vendor's own office is discouraged.
```

Comment:
```suggestion
Public events for the project must also be handled in a vendor-neutral way. Even in cases where the CNCF events staff is not engaged, all potential vendors must have an equal opportunity to sponsor and participate in the event. For this reason, hosting an project-branded event at any specific vendor's own office is discouraged.  The KCD program [has an explanation](https://github.com/cncf/kubernetes-community-days/blob/main/committee-resources/content-management.md) of what vendor neutral means for KCDs, which should be considered to apply to all official CNCF community events.
```

Reply:
Addition from the Googledoc; didn't realize you hadn't seen it before drafting the PR.

Reply:
Sorry missed that